### ISSUE TITLE
Cannot show query result.

### ISSUE THREAD

**Issue by leeyaunlong** [EINDEX/logseq-copilot#28]
Chrome:102, copilot : 1.11.3
Save is work, icon will show correct saved item counts.
But click the icon, cannot show query result.

![snap_20230508_160438_chrome](https://user-images.githubusercontent.com/19405648/236770264-d5ed118c-0a21-4951-8aaf-93f8b77793a2.png)

I checked service work log:
![snap_20230508_155733_chrome](https://user-images.githubusercontent.com/19405648/236769920-fef9f7be-9dc2-41b8-a017-9efe8110f134.png)

**Comment by EINDEX** [comment]
Do you mind sharing block content here?

**Comment by leeyaunlong** [comment]
since update to 1.11.3, the copilot will show some random issue.
- cannot show capture page's content in popup window(like above screen shot)


- the link cannot jump to logseq in popup window (in sometime will happen, but link is good, i can run it in window).
for this case, the 4 links cannot jump to logseq, but it always workfine with old version.
![snap_20230509_090712_chrome](https://user-images.githubusercontent.com/19405648/236968676-eb61f957-08d4-4ff1-b933-6b8accf4a9d6.png)

**Comment by leeyaunlong** [comment]
> Do you mind sharing block content here?
The url schema jump to logseq function work well at v1.11.0
Since v1.11.1 , thejump function failed.

**Comment by leeyaunlong** [comment]
> Do you mind sharing block content here?
If I right click the "to block" or "note" , select open in new tab , the copilot will trigger the logseq open correct location.

I guess the link added some custom event , cause the issue.

Final I find out the popup source :
```
        document.querySelectorAll('a').forEach((e) => {
          e.onclick = () => {
            connect.postMessage({
              type: 'open-page',
              url: e.href,
            });
            window.close();
          };
```

If I remove the click event at those type link , will work well.

**Comment by mbm-peter** [comment]
I was just looking at this and saw that the 'open-page' message was logged by [this line](https://github.com/EINDEX/logseq-copilot/blob/25438597bb9813e78a681a0be007b3647ab8cb73/src/pages/background/index.ts#L24), instead of the other message handler. Perhaps this helps with debugging the problem further?

**Comment by EINDEX** [comment]
Thanks for your information, which help me fix this issue!

The fix version on v1.11.4 will release on the store soon.
